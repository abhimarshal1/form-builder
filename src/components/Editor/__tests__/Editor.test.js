import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Components
import { SchemaEditor } from '..';

const renderer = new ShallowRenderer();

describe('SchemaEditor', () => {
  test('shallow render & match the snapshot', () => {
    renderer.render(<SchemaEditor name="editor" />);

    const output = renderer.getRenderOutput();
    expect(output).toMatchSnapshot();
  });
});
