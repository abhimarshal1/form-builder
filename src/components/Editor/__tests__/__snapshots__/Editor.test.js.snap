// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SchemaEditor shallow render & match the snapshot 1`] = `
<Memo(Editor)
  defaultLanguage="json"
  defaultValue="// some comment"
  height="100%"
  name="editor"
  options={
    Object {
      "fontSize": "16",
    }
  }
  theme="vs-dark"
/>
`;
